R/metareactive.R

```diff
@@ -327,7 +311,7 @@ metaExpr_ <- function(expr, env = parent.frame(), quoted = FALSE, localize = "au
   # metaExpr() moves us from mixed to meta state
   withMetaMode(mode = TRUE, {
     expr <- comment_flags(expr)
-    expr <- expandExpr(expr, if (topLevelDynVars) .globals$dynamicVars, env)
+    expr <- expandExpr(expr, list(), env)
```

Comment:
I think this was the only place we were passing that second argument to `expandExpr`--if that's so, I'd prefer to just remove that parameter altogether.